keyboards/yampad/yampad.c

```diff
@@ -0,0 +1,62 @@
+    
+/* Copyright 2019
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "yampad.h"
+
+__attribute__((weak))
```

Comment:
The OLED functions should be wrapped in an `#if defined` statement, so users who wish to skip the OLED can still compile their firmware.

```suggestion
#if defined(OLED_DRIVER_ENABLE)
__attribute__((weak))
```